src/main/java/com/google/maps/GeoApiContext.java

```diff
@@ -53,8 +54,15 @@
   private long errorTimeout = DEFAULT_BACKOFF_TIMEOUT_MILLIS;
 
   public GeoApiContext() {
+    this(null);
+  }
+
+  public GeoApiContext(Proxy proxy) {
```

Comment:
Instead of setting the proxy in the constructor, please move it to be set in line the builder pattern we use for the other properties.